---a/net/minecraft/world/border/EnumBorderStatus.java
+++b/net/minecraft/world/border/EnumBorderStatus.java
@@ -1,10 +1,6 @@
 package net.minecraft.world.border;
 
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
 
-public enum EnumBorderStatus
-{
+public enum EnumBorderStatus {
     GROWING(4259712),
     SHRINKING(16724016),
     STATIONARY(2138367);
@@ -12,9 +12,7 @@
 
     private static final String __OBFID = "CL_00002013";
 
-    private EnumBorderStatus(int id)
-    {
+    private EnumBorderStatus(int id) {
         this.id = id;
     }
-
-    @SideOnly(Side.CLIENT)
-    public int getID()
-    {
-        return this.id;
-    }
-}
+}
